lib/buffer.js

```diff
@@ -264,7 +264,8 @@ function fromObject(obj) {
   }
 
   if (obj) {
-    if (isArrayBuffer(obj.buffer) || 'length' in obj) {
+    if (isArrayBuffer(obj.buffer) || 'length' in obj
+        || isSharedArrayBuffer(obj)) {
```

Comment:
Although I prefer this style as well, this project uses the operators on the right-hand side:

```js
    if (isArrayBuffer(obj.buffer) || 'length' in obj ||
        isSharedArrayBuffer(obj)) {
```